client/ayon_fusion/api/plugin.py

```diff
@@ -205,6 +209,18 @@ def _configure_saver_tool(self, data, tool, product_name):
             .replace("{task}", "{task[name]}")
         )
 
+        host_name = get_current_host_name()
+        project_name = self.create_context.get_current_project_name()
+        extra_data = template_data.get_template_data_with_names(
+            project_name,
+            data["folderPath"],
+            data["task"],
+            host_name
+        )
```

Comment:
This would be more efficient if we could use the create context's cache potentially to get the entity data, etc.

I'll try to push a tweak to do so.